src/server.cpp

```diff
@@ -594,7 +594,8 @@ void Server::AsyncRunStep(bool initial_step)
 		// Run Map's timers and unload unused data
 		ScopeProfiler sp(g_profiler, "Server: map timer and unload");
```

Comment:
Nope, that would be outside of the diff, make the actual changes of the commit less readable, possibly create merge conflicts, and make life harder for forkers.